api/util/registerUser.js

```diff
@@ -0,0 +1,127 @@
+const User = require('../models/User');
+const config = require('../config/config');
+
+/**
+ * Register a new user.
+ * @param {Object} newUser - The user that is to be registered
+ * @param {(Object} userToAdd - The information about the user that is
+ *                              registering
+ */
```

Comment:
i think this function returns something, so document it
```suggestion
 * @returns {Object} blah blah blah
 */
```